src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -14,7 +14,26 @@
 @ApiController('rgw')
 @AuthRequired()
 class Rgw(RESTController):
-    pass
+
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def status(self):
+        status = {'available': False, 'message': None}
+        try:
+            instance = RgwClient.admin_instance()
+            # Check if the service is online.
+            if not instance.is_service_online():
+                status['message'] = 'Failed to connect to the RGW Admin Ops API.'
```

Comment:
Textual improvement suggestion:

_Failed to connect to the object gateway's Admin Ops API._